src/Google.Api.Gax.Grpc/ClientHelper.cs

```diff
@@ -69,5 +70,27 @@ public ClientHelper(ServiceSettingsBase settings)
                 .WithRetry(Clock, Scheduler)
                 .WithUserAgent(_userAgent);
         }
+
+        /// <summary>
+        /// Builds an <see cref="ApiBidirectionalStreamingCall"/> given a suitable underlying duplex call.
+        /// </summary>
+        /// <typeparam name="TRequest">Request type, which must be a protobuf message.</typeparam>
+        /// <typeparam name="TResponse">Response type, which must be a protobuf message.</typeparam>
+        /// <param name="grpcCall">The underlying gRPC duplex streaming call.</param>
+        /// <param name="perMethodCallSettings">The default method call settings.</param>
+        /// <param name="streamingSettings">The default streaming settings.</param>
+        /// <returns>An API call to proxy to the RPC calls</returns>
+        public ApiBidirectionalStreamingCall<TRequest, TResponse> BuildApiCall<TRequest, TResponse>(
+            Func<CallOptions, AsyncDuplexStreamingCall<TRequest, TResponse>> grpcCall,
+            CallSettings perMethodCallSettings,
+            BidirectionalStreamingSettings streamingSettings)
+            where TRequest : class, IMessage<TRequest>
+            where TResponse : class, IMessage<TResponse>
+        {
+            // TODO: What about retry?
```

Comment:
Looks like retry is basically not an option - so maybe we should check that there *is* no retry?